Feature/gcs plugin sanitizing ipynb file

srinivasan-acn opened the pull request:
Implemented notebook file sanitization using nbformat ( version 4 ) library.

ptch314 commented on gcs_jupyter_plugin/controllers/gcs.py:
```diff
@@ -0,0 +1,282 @@
+
+                self.finish(file_content)
+                
+                # if format == 'text':
```
remove commented out code

srinivasan-acn replied:
Noted. Thanks for pointing this out @ptch314. This change is applied in the latest PR ( PR#16), FYI.

Note : We've applied this change in the latest branch to avoid cascading updates through our stacked PRs. Going forward, new branches will be directly created from the fork main to prevent this issue.

ojarjur reviewed (commented):
Please add tests to this PR.

I would like to see something that verifies that the trusted bit is cleared out when loading the ipynb file.

ptch314 reviewed (approved):
Approved pending addressing Omar's comment about adding tests.

srinivasan-acn commented:
@ojarjur @ptch314 , I have addressed your comment and the changes are which is part of PR#18 https://github.com/GoogleCloudDataproc/gcs-jupyter-plugin/pull/18
